scipy/optimize/_differentialevolution.py

```diff
@@ -132,6 +134,29 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         ``np.std(pop) <= atol + tol * np.abs(np.mean(population_energies))``,
         where and `atol` and `tol` are the absolute and relative tolerance
         respectively.
+    updating : str, optional
+        Should be one of:
+
+            - 'immediate'
+            - 'deferred'
+
+        If `immediate` the best solution vector is continuously updated within
+        a single generation. This can lead to faster convergence as trial
+        vectors can take advantage of continuous improvements in the best
+        solution.
+        With `deferred` the best solution vector is updated once per
+        generation. Only `deferred` is compatible with parallelization, and the
+        `workers` keyword can over-ride this option.
```

Comment:
would be good to add `.. versionadded:: 1.2.0` to both new keywords

Reply:
I didn't know that one could add that markup to keywords. Does it need a new line before/after?

Reply:
blank line first and then the directive IIRC, no blank line below